config/default/core.extension.yml

```diff
@@ -38,6 +38,8 @@ module:
   entity_browser: 0
   entity_browser_entity_form: 0
   entity_embed: 0
+  entity_print: 0
+  entity_print_views: 0
```

Comment:
Do we want `entity_print_views` enabled for default too? I'm guessing that this isn't necessary for what we are trying to accomplish with this PR.

Reply:
if we can avoid turning on the module to rid us of the admissions issue, i'd vote for that